invoice/invoice.go

```diff
@@ -36,11 +36,14 @@ func CreateInvoice(ctx context.Context, client *model.Odoo, invoice invoice.Invo
 			line := opts.invoiceLineDefaults
 			line.CategoryID = categoryID
 
-			total := math.Round(item.Total*100) / 100
-			line.Name = fmt.Sprintf("%s Qty: %.2f %s, PPU: %.10f, Disc: %.0f%%",
-				item.Description, item.Quantity, item.Unit, item.PricePerUnit, item.Discount*float64(100))
+			name, err := opts.ItemDescriptionRenderer().RenderItemDescription(ctx, item)
+			if err != nil {
+				return 0, fmt.Errorf("error rendering line template: %w", err)
```

Comment:
I would love to have a bit more human readable error here or a layer down. Most importantly: If a new product is added and we inevitably forget to add the corresponding template we will get a `template: no template "foo" associated with template "root"` error.

That error is very clear for you now, but probably not so much for someone else in a year..

Reply:
Yep. I'll use `template.Lookup()` and return a nice error message 👍